src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -61,7 +62,7 @@ internal PeekDefinition(ServerConnection serverConnection, ConnectionInfo connIn
             Initialize();
         }
 
-        private Database Database
+        internal Database Database
```

Comment:
In this scenario, a new peek definition call would be made after the use tempdb execution and that will have a new peek definition object.